float64[] times
geometry_msgs/Pose[] poses
geometry_msgs/Twist[] vels